### PR TITLE
SSL4EO-L MoCo-v3 Experiment

### PR DESCRIPTION
**Pull request by isaaccorley**

### CODE REVIEW

File: conf/ssl4eo-l.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+module:
+  _target_: torchgeo.trainers.MoCoTask
+  model: resnet50
+  weights: True
```
Comment (adamjstewart) [inline conf/ssl4eo-l.yaml:4]: Do we want to start from ImageNet weights?

File: conf/ssl4eo-l.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+module:
+  _target_: torchgeo.trainers.MoCoTask
+  model: resnet50
```
Comment (adamjstewart) [inline conf/ssl4eo-l.yaml:3]: Ideally we'll want pre-trained models for ResNet-18, ResNet-50, and ViT-S. Would probably prioritize the last 2. If we don't have any ViTs I think the reviewers will laugh at us.

File: conf/ssl4eo-l.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  gather_distributed: True
+  size: 224
+  grayscale_weights: null
+
+datamodule:
+  _target_: torchgeo.datamodules.SSL4EOLDataModule
+  root: /workspace/storage/data/ssl5eo_l_etm_toa
+  split: etm_toa
+  seasons: 2
+  batch_size: 256
```
[the remark this answers is not in the document]
Comment (isaaccorley) [reply conf/ssl4eo-l.yaml:24]: I haven't read the v3 paper in awhile, but I'm curious if the memory bank (MoCo-v2 settings) is better in the case you can't push the batch size to 4096, the memory bank will still allow you to sample many negatives

File: conf/ssl4eo-l.yaml (hunk 2 above)
Comment (isaaccorley) [reply conf/ssl4eo-l.yaml:3]: I think we should focus on ResNet50 and ViT-S first. The only other SSL paper I've seen recently that trains ResNet18 is SeCo.

File: conf/ssl4eo-l.yaml (hunk 1 above)
Comment (isaaccorley) [reply conf/ssl4eo-l.yaml:4]: `True` should use imagenet weights. See [link](https://github.com/microsoft/torchgeo/blob/3196dab4dbb9df15d47f266e0fa2c7ec6eb415a3/torchgeo/trainers/moco.py#L227)

File: conf/ssl4eo-l.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (adamjstewart) [reply conf/ssl4eo-l.yaml:24]: Yeah, not sure either. Let's stick to vanilla v3 for now. We can add memory bank later if necessary.

**Comment by adamjstewart** [comment]
Do we still need this PR?

**Comment by adamjstewart** [comment]
Ping @isaaccorley

**Comment by adamjstewart** [comment]
None of these files even exist anymore, I think we can close this.

**Pull request closed without merging**